CONTRIBUTING.rst

```diff
@@ -163,6 +163,7 @@ their individual contributions.
 * `Alex Stapleton <https://www.github.com/public>`_
 * `Alex Willmer <https://github.com/moreati>`_ (alex@moreati.org.uk)
 * `Ben Peterson <https://github.com/killthrush>`_ (killthrush@hotmail.com_)
+* `Buck Evan, copyright Google LLC <https://github.com/bukzor>`_ 
```

Comment:
I think this is fine as long as @bukzor has filled out the necessary internal paperwork RE open source contributions, which we can take his word on.

Reply:
Never having worked at Google, I'll take your word for it 😀

@bukzor, that means it's just the test and whitespace issue to go 😄

Reply:
I'm an agent of the company in this respect. I verify that I put my copyrighted work under your license just as your other contributors do.

Strictly FYI: Because this project is under MPL, it falls into the category of "no approval necessary" projects. Google only asks that I use my @google.com email for commits, and any contributor-copyright is labelled as "Google LLC".

Reply:
@Zac-HD: Github says you've requested changes, but the "view changes" button 404s for me. I don't think I can get out of this state without your help.

Reply:
Neat - it's always nice to hear about sensible IP policies!

I suspect rebasing broke the link; Github really seems to discourage it 😢 (or they just can't be bothered supporting it...).  You've made all the changes I'd request now anyway ✨